docs/developer/decision-records/README.md

```diff
@@ -43,3 +43,4 @@
 - [2022-10-20 Trust Framework Adoption repository](2022-10-20-trust-framework-adoption-repository)
 - [2022-10-21 Gradle Version Catalogs](2022-10-21-gradle-versioncatalogs/)
 - [2022-10-31 Service Layer](2022-10-31-aggregate-service-layer)
+- [2022-11-09 API Restructuring](2022-11-09-api-refactoring)
```

Comment:
This does not work as the path/ automatically points to a `README.md`. Pls refer to a document and within this provide a link to the other one. Or create a README.md that lists the links to both files.

Reply:
good to know!